runtime/src/main/java/org/corfudb/runtime/exceptions/unrecoverable/UnrecoverableCorfuInterruptedError.java

```diff
@@ -4,24 +4,21 @@
 
 /** Thrown when a Corfu Client/Server is interrupted in an unrecoverable manner.
  *  Once interrupted in this manner, Corfu can only guarantee safety if the application
- *  exits, as Corfu safe is no longer recoverable.
+ *  exits, as Corfu safety is no longer recoverable.
  */
 public class UnrecoverableCorfuInterruptedError extends UnrecoverableCorfuError {
 
     /** Construct a new {@link UnrecoverableCorfuInterruptedError} with a root cause.
      *
-     * <p> Also resets the interrupt pending flag.
      * @param cause     The root cause of the exception.
      */
     public UnrecoverableCorfuInterruptedError(@Nonnull InterruptedException cause) {
         super(cause);
-        Thread.currentThread().interrupt();
     }
 
     /** Construct a new {@link UnrecoverableCorfuInterruptedError} with a given message
      * and root cause.
      *
-     * <p> Also resets the interrupt pending flag.
```

Comment:
but the constructor does Thread.currentThread().interrupt()

Reply:
@medhavidhawan Oops missed that. Fixed.